### ISSUE TITLE
Extra newlines added to block comments

### ISSUE THREAD

**Issue by yebblies** [Hackerpilot/dfmt#20]
Only occurs with windows line endings, the `\r\n` gets replaced with `\r\n\n` inside the block comment.
```d
SignExtendedNumber opMul(const SignExtendedNumber a) const
 {
     // perform *saturated* multiplication, otherwise we may get bogus ranges
     //  like 0x10 * 0x10 == 0x100 == 0.
     /* Special handling for zeros:
     */
}
```
```d
SignExtendedNumber opMul(const SignExtendedNumber a) const
 {
     // perform *saturated* multiplication, otherwise we may get bogus ranges
     //  like 0x10 * 0x10 == 0x100 == 0.
     /* Special handling for zeros:

     */
}
```

**Comment by Hackerpilot** [comment]
WIIIIINDOOOOOWS!

**Comment by Hackerpilot** [comment]
Fixed in 51b991f50f643e1a5afe11e522d731b1178b953c
